cobaya/samplers/mcmc/mcmc.py

```diff
@@ -260,6 +262,10 @@ def set_proposer_blocking(self):
                                  "Use 'oversample_power' to control the amount of "
                                  "dragging steps.")
             # END OF DEPRECATION BLOCK
+            if self.temperature:
+                raise LoggedError(
+                    self.log,
+                    "Temperature != 1 and dragging are not compatible at the moment.")
```

Comment:
At a quick glance, I think dragging is correct with just the current PR changes (since temperature just scales all the logposts by the same linear factor)?

Reply:
It should, I guess, for any sampling algorithm, since it's just importance reweighting? Anyway, will test before merging with baseline Planck.